### PR TITLE
Fix speaker's dialog on session details. Fixed #374

### LINKED ISSUE #374: v2: scrollbar dissapears / page is locked
There is bug in scenario, when closing dialog with [X]

How to reproduce:
- Open schedule
- Select session
- Navigate to speaker
- Close dialog with [X]
after close: empty dialog appears, scrolling on the page is blocked (scrollbar dissapears), only refresh helps.

### PR DESCRIPTION
**Pull request by ozasadnyy**

### CODE REVIEW

File: src/elements/dialogs/session-details.html [hunk 1]
```diff
@@ -171,7 +175,7 @@ <h3>{$ sessionDetails.speakers $}</h3>
         return getComputedStyle(this, `--${this.generateClassName(value)}`);
       }
 
-      _getCLoseBtnIcon(isLaptopViewport) {
+      getCloseBtnIcon(isLaptopViewport) {
```
Comment (sophieH29) [inline src/elements/dialogs/session-details.html]: Why have you removed underscore?

Comment (ozasadnyy) [reply src/elements/dialogs/session-details.html]: My mistake. I was changing `..CLose..` to `..Close..`

Comment (sophieH29) [reply src/elements/dialogs/session-details.html]: ok :) just rename it, and you can merge. Going to approve

File: src/elements/dialogs/session-details.html [hunk 2]
```diff
@@ -156,6 +156,10 @@ <h3>{$ sessionDetails.speakers $}</h3>
 
       _close() {
         dialogsActions.closeDialog(DIALOGS.SESSION);
+        this.dispatchEvent(new CustomEvent('reset-query-params', {
```
Comment (sophieH29) [inline src/elements/dialogs/session-details.html]: How does it work? Where is this event listened to?

Comment (sophieH29) [inline src/elements/dialogs/session-details.html]: Where is this event listened to?

Comment (ozasadnyy) [reply src/elements/dialogs/session-details.html]: Here is a listener: https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/hoverboard-app.html#L341

**Pull request merged**